### DISCUSSION TITLE
DeliveryChannels - Name Default

### DISCUSSION THREAD

**Discussion by BAY-GOFNX** [aws-solutions/workload-discovery-on-aws#468] [General]
Hi All,

Is it possible in the solution when create the deliveryChannel to change the name default for another? If yes, can you teach me what are the files necessary to be changed for to do that?

Regards,

**Comment by svozza** [comment]
This is not really Workload Discovery question. However, as per the [AWS Config docs](https://docs.aws.amazon.com/config/latest/developerguide/manage-delivery-channel.html#update-dc-rename), if you want to change the name of the delivery channel you must first delete the old one and recreate the new one with a new name.

**Reply by BAY-GOFNX** [reply]
Is it possible to have 2 delivery channels?? or only one?

**Reply by svozza** [reply]
No, as per the document I linked above:

> You can have only one delivery channel per region per AWS account, and the delivery channel is required to use AWS Config.

**Reply by BAY-GOFNX** [reply]
ok, thank you

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
